scripts/ultra_quick_test.py

```
@@ -33,18 +33,40 @@ def test_compensation():
+        print(f"   ❌ Ошибка: {e}")
+        return False
+
+def test_adaptive_filters():
+    """Тест адаптивных фильтров"""
+    try:
+        result = subprocess.run(
+            [sys.executable, "test/test_adaptive_filters.py"],
+            capture_output=False, text=True, timeout=10, cwd=Path(__file__).parent.parent
+        )
+        return result.returncode == 0
     except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)